src/Raven.Server/Documents/Commands/Indexes/TestIndexCommand.cs

```diff
@@ -0,0 +1,52 @@
+using System.Net.Http;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Http;
+using Raven.Client.Json;
+using Raven.Server.Documents.Indexes.Test;
+using Sparrow.Json;
+
+namespace Raven.Server.Documents.Commands.Indexes;
+
+internal sealed class TestIndexCommand : RavenCommand<BlittableJsonReaderObject>
```

Comment:
I would prefer to have here a managed class instead of a blittable.

Reply:
Our response has to be generic type, because fields we return as `TestIndexResult` object (e.g. `IndexEntries`) are generic, they vary depending on data provided by user and used collections.